.github/CONTRIBUTING.md

```diff
@@ -43,15 +46,49 @@ npm run lint-md
 npm run lint-less
 ```
 
+## Conventional Commits
+
+We use [Conventional Commits][] as our commit message format. This leads to more
+uniform messages in the project history.
+
+### Type
+
+We use [the change types Angular uses][].
+
+
+### Scope
+
+We have not yet formalized scopes. In practice scopes used so far include:
+
++ `changelog`
++ `ci`
```

Comment:
:question: ci is already a type, do we also want it to be a scope?

Reply:
Nope. :)